lighthouse-core/scripts/legacy-javascript/README.md

```diff
@@ -0,0 +1,8 @@
+Run:
+
+```sh
+yarn install
+node run.js
+```
+
+Digging into core-js: https://gist.github.com/connorjclark/cc583554ff07cba7cdc416c06721fd6a
```

Comment:
Could we get a little more explanation of what this directory does and how to use it? :)

I'm thinking... 
- What do the stats in summary-signals and summary-stats represent
- how do we interpret them
- do they need to be updated and asserted like lantern snapshots or sample json
- what would future improvements be doing (adding new core-js pollyfills and testing out new signals? finding real world bundles and checking them in?)
- what does regression/improvement look like when someone does that

Reply:
> do they need to be updated and asserted like lantern snapshots or sample json

In CI? I don't think this is necessary, but we could do it. `yarn test-legacy-javascript` just before the `git diff --exit-code`, and skip if no files in `scripts/legacy-javascript` / `audits/legacy-javascript.js` were changed.

Reply:
Right - this audit is checking for the detection of particular build processes (babel / corejs), so it's necessary to have a corpus for debugging / improvement. but the corpus is too large for our CI IMO, and the results shouldn't change except for explicit changes to this audit (for which, one would use this tool and update the summaries for review in PR).

we could do a guarded test like lantern, might be cool, wdyt?

Reply:
> we could do a guarded test like lantern, might be cool, wdyt?

Yeah SGTM 👍